addons/l10n_il/data/account_data.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo noupdate="1">
+
+    <record id="tax_group_vat_16" model="account.tax.group">
+        <field name="name">VAT 16%</field>
+    </record>
```

Comment:
@jcr-odoo Wikipedia (https://en.wikipedia.org/wiki/Taxation_in_Israel ) tells me the VAT is 17% and 16% is a really old one.  I also see there are no fiscal positions defined, while there is a clear distinction between export (Exempt) and the inland.

Reply:
There is one tax that is 16%, its for palestina, partner approved wwith testing what is developed, so its good to go.

Reply:
Here we could also have fiscal positions to make it easier for the user.